sci-libs/galib/galib-2.4.7-r2.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+MY_PV="${PV//\./}"
+
+DESCRIPTION="Library for genetic algorithms in C++ programs"
+HOMEPAGE="http://lancet.mit.edu/ga/"
+SRC_URI="http://lancet.mit.edu/ga/dist/galib${MY_PV}.tgz"
+
+LICENSE="BSD examples? ( GPL-2 )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc examples"
+
+S="${WORKDIR}/${PN}${MY_PV}"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-2.4.7-fix-buildsystem.patch"
+	"${FILESDIR}/${PN}-2.4.7-fix-c++14.patch"
+	"${FILESDIR}/${PN}-2.4.7-Wformat-security.patch"
+)
+
+src_prepare() {
+	default
+	sed -e "s:/include:${EPREFIX}/usr/include:" \
+		-e "s:/lib:${EPREFIX}/usr/$(get_libdir):" \
+		-i makevars || die
+	find ga/ -type f -exec sed -i -e "s/register int/int/g" {} \; || die
```

Comment:
Did you see my comments on a previous PR about sed? I'm really not a fan of it for stuff like this.

(If you can't find it, let me know,a nd I'll dig it up.)